main_flow/main_flow.py

```diff
@@ -57,24 +58,14 @@ def loadCache():
     return model_instruction and model_ingredients and top_instru_dict and top_ingri_dict
 
 
-def load_from_word_to_steam():
-    try:
-        with open('data.pkl', 'rb') as handle:
-            global from_word_to_steam_cache
-            from_word_to_steam_cache = pickle.load(handle)
-    except Exception:
-        logging.error('load_cache error')
-        from_word_to_steam_cache = {}
-
-
 def periodically_save_cache():
     while True:
         # Wait for new data to accumulate
         time.sleep(CYCLE_TIME_TO_SAVE_CACHE)
 
         # Save to disk
-        with open('data.pkl', 'wb') as handle:
-            pickle.dump(from_word_to_steam_cache, handle, protocol=pickle.HIGHEST_PROTOCOL)
+        with open('../stemming/data.pkl', 'wb') as handle:
+            pickle.dump(from_word_to_stem_cache, handle, protocol=pickle.HIGHEST_PROTOCOL)
```

Comment:
I think this will break the linux server

Reply:
i moved data.pkl from mainflow to stemming and this path changed automatically with refactor
i will look into this..

Reply:
why would it break the server? is that path format for windows?>

Reply:
ok sorry I see you moved the file
but please check out this method
def periodically_save_cache():
    while True:
        # Wait for new data to accumulate
        time.sleep(CYCLE_TIME_TO_SAVE_CACHE)

        # Save to disk
        with open('data.pkl', 'wb') as handle:
            pickle.dump(from_word_to_steam_cache, handle, protocol=pickle.HIGHEST_PROTOCOL)

make sure we are saving it in the new location

Reply:
ohh ok my mistake...
BUT 
why this method is still in main_flow?
it's related only to stem - c0an you please open a ticket to move it to stemming?(or do it as a new commit?